src/OSPSuite.Presentation/Presenters/Importer/MappingParameterEditorPresenter.cs

```diff
@@ -95,7 +95,7 @@ public void SetErrorTypeOptions(IEnumerable<string> types, string selected)
          _errorEditorPresenter.SetOptions(new Dictionary<string, IEnumerable<string>>() { { "", types } }, selected);
          View.ShowErrorTypes();
 
-         if ( selected.Equals(Constants.STD_DEV_GEOMETRIC))
+         if (selected!= null && selected.Equals(Constants.STD_DEV_GEOMETRIC))
```

Comment:
In general you can do Equals(selected,xxx). If selected is nulk, it will return false